src/genomic_features/__init__.py

```diff
@@ -1,5 +1,7 @@
 from importlib.metadata import version
 
+from . import ensembl
+from .annotate_anndata import annotate_anndata
 from . import ensembl, filters
```

Comment:
```suggestion
from .annotate_anndata import annotate_anndata
from . import ensembl, filters
```

Ensembl should already be handled by `from . import ensembl, filters`